core/include/mm/sp_mem.h

```diff
@@ -0,0 +1,59 @@
+/* SPDX-License-Identifier: BSD-2-Clause */
+/*
+ * Copyright (c) 2021, Arm Limited.
+ */
+
+#ifndef __MM_SP_MEM_H
+#define __MM_SP_MEM_H
+
+#include <ffa.h>
+#include <mm/fobj.h>
+#include <sys/queue.h>
+
+struct sp_mem_receiver;
+struct sp_mem;
+
+struct sp_mem_receiver {
```

Comment:
Please describe these structs, right now it's not very easy to see how everything connects.

Reply:
Done